### ISSUE TITLE
Recruitment issue

### ISSUE THREAD

**Issue by marxopolis** [NationStates/NationStatesPlusPlus#27]
I have been using an auto-telegram from nationstates++ for my region, New Socialist Bloc, the header (that says who you are telegramming) has been the same for almost a week "Telegramming: Godfreaks".

i have deleted the campaigns, made new ones, uninstalled NS++, re-installed it, and it still does this. It is not just the display, over the time this has been happening there have been zero sent telegrams.

**Comment by Eluvatar** [comment]
Are you continuing to experience problems?

If yes, the same problems or different ones?

**Comment by marxopolis** [comment]
yes, the same problems, i just restarted the recruitment program to check, it is still stuck trying to telegram the same person.
